File: js/p6.js
var _ = require('underscore');

_.mixin({
    sum : function(seq) {
        return _.reduce(seq, function(memo, num) { return memo + num; }, 0);
    }
});

var r = _.range(1, 101);

var sum_of_square = _.sum(_.map(r, function(x) {return x*x;}));
var square_of_sum = Math.pow(_.sum(r), 2);
var rslt = square_of_sum - sum_of_square;

console.log(rslt);

File: go/p6.go
package main

import (
    "fmt"
    "math"
)


func main() {
    sum_of_squre := 0
    sum := 0
    for i := 1; i < 101; i++ {
        sum += i
        sum_of_squre += i*i
    }

    rslt := int(math.Pow(float64(sum), 2)) - sum_of_squre
    fmt.Println(rslt)
}
